eng/feeds/azure-scaffolding.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Should also have a PR updating the doc in `cadl-azure` as the existing link will be broken